index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <img src="images.svg\image_geometry_1 (1).svg" alt="" class="top-left">
+        <img src="images.svg\ic_menu (1).svg" alt="" class="hamburger">
+    </header>
```

Comment:
- [ ] As required for this project your logo should use `nav` tag for semantically reason but what i can see here you just use the header tag and `img` tag kindly take care of this for your project to match all requirements :wink:

Reply:
but I used header , isn't that semantic?

Reply:
> * [ ]  As required for this project your logo should use `nav` tag for semantically reason but what i can see here you just use the header tag and `img` tag kindly take care of this for your project to match all requirements 😉

but I used header , isn't that semantic?

Reply:
- [ ] As required for this project The hamburger button is on the right, and has a `nav` tag kindly fix this for your project to look professional and for semantic reason :wink: